lib/CreateChangesetUpdateChangelog.js

```diff
@@ -5,21 +5,49 @@ const replace = require('replace-in-file');
 module.exports = class {
     constructor() {
     }
-   createChangeset(typeOfMigration,dbChange,changesetPath,version){ 
+   verifyifChangesetExists(changeset,changelogPath){
+        var path=changelogPath.split("/")
+        let changelogDir=path[0]+"/"
+        for(var i=1;i<path.length-2;i++){
+            changelogDir+=path[i]+"/"
+        }
+        changelogDir+="sql"
```

Comment:
I am not sure what's trying to achieve here ... 
Maybe you are trying with same thing that '**path.dirname()**' does which returns one level up from the path I think.
Also, could you use path.join(somePath, 'sql') instead?

Reply:
I am not sure this would help in this case, since i need to do a fair amount of processing to get the end result